.circleci/config.yml

```diff
@@ -127,3 +215,20 @@ workflows:
       - test-python-3.5-uvloop
       - test-pypy-3.6-asyncio
       - test-pypy-3.5-asyncio
+  docker:
+    jobs:
+      - build-docker:
+          filters:
+            branches:
+              only: /.*/
```

Comment:
Why a filter that matches everything? :slightly_smiling_face:

Reply:
Because tags are not included by default (and hoping this works for tags).

Reply:
I don't think a branch filter will match tags. You need the `tags` filter for that IIRC.

Reply:
:ok: